### DISCUSSION TITLE
Improve the UX of the editor

### DISCUSSION THREAD

**Discussion by chrisvidal** [OFFLINE-GmbH/oc-boxes-support#174] [Ideas and feature requests]
It would be great if the title of the label `add a child to this box` could be contextual.
that would help tremendously to know what we are adding to who

we could reuse the name of the parent box per default.
it would look like:
`add a child to this Grid`
`add a child to this Container`
etc...

![image](https://github.com/user-attachments/assets/a4c2193b-88e0-45dd-872a-2937cf402361)

**Comment by tobias-kuendig** [comment]
Great suggestion! I'll add this to the backlog.

**Reply by tobias-kuendig** [reply]
Added in v3.6.45, thanks!

**Reply by chrisvidal** [reply]
what a delight to use now !
